components/Opportunities.tsx

```diff
@@ -0,0 +1,106 @@
+"use client"
+import React, { useState, useEffect } from "react"
+import Link from "next/link"
+import { LinkCard } from "@/components/LinkCard"
+import {
+  MapPinIcon,
+  ArrowRightIcon,
+  UserPlusIcon,
+} from "@heroicons/react/24/solid"
+
+interface OpportunityProps {
+  total: number
+  jobPostings: PositionProps[]
+  facets: Object[]
+  userAuthenticated: boolean
+}
+
+interface PositionProps {
+  title: string
+  externalPath: string
+  locationsText: string
+  postedOn: string
+  bulletFields: string[]
+}
+
+export const Opportunities: React.FC = () => {
+  const initialData = {
+    total: 0,
+    jobPostings: [],
+    userAuthenticated: false,
+    facets: [],
+  }
+  const [data, setData] = useState<OpportunityProps>(initialData)
+  const [isLoading, setLoading] = useState(true)
+  const [error, setError] = useState(null)
+
+  const handleApiCall = async () => {
+    try {
+      const res = await fetch("/api/about", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+      })
+
+      if (!res.ok) {
+        throw new Error(`Failed to fetch: ${res.status}`)
+      }
+
+      const opportunities = await res.json()
+      setData(opportunities.json)
+    } catch (err: any) {
+      setError(err.message)
+    } finally {
+      setLoading(false)
+    }
+  }
+  useEffect(() => {
+    handleApiCall()
+  }, [])
+
+  return (
+    <section>
+      <Link href="#opportunities" className="text-neutral-900">
+        <h2 className="text-3xl bg-primary-500 text-white p-4 flex items-center gap-2">
+          <UserPlusIcon className="h-7 w-7" />
+          Opportunities
+        </h2>
+      </Link>
+      {isLoading && <p className="px-2 font-bold">Loading...</p>}
```

Comment:
For RIDDC stuff, I recently learned that making a loading spinner is actually really easy! It might be worth looking into if you want keep the loading logic in here. I can link you the code I used for it if you want.